test/points.jl

```diff
@@ -1,44 +1,37 @@
-using Geodesy
-using Base.Test
+@testset "Points and distances" begin
+    # Construction
 
-# Construction
+    x, y = (rand(2) - .5) * 10_000
 
-x, y = (rand(2) - .5) * 10_000
+    @test ENU(x, y) == ENU(x, y, 0.0)
 
-@test ENU(x, y) == ENU(x, y, 0.0)
+    @test LLA(x, y) == LLA(x, y, 0.0)
 
-@test LLA(x, y) == LLA(x, y, 0.0)
+    ECEF(x, y, 0.0)
 
-ECEF(x, y, 0.0)
+    # Distance
 
-# get* methods
+    @test distance(ENU(1, 1, 1), ENU(2, 2, 2)) == sqrt(3)
 
-lla = LLA(y, x, rand())
-@test LLA(getY(lla), getX(lla), getZ(lla)) == lla
+    @test distance(ECEF(1, 1, 1), ECEF(3, 3, 3)) == sqrt(12)
 
-enu = ENU(x, y, rand())
-@test ENU(getX(enu), getY(enu), getZ(enu)) == enu
+    randLLA() = LLA((rand() - .5) * 180,
+                    (rand() - .5) * 360,
+                    (rand() - .5) * 18_000)
```

Comment:
You've got `randLLA()` elsewhere, should you use that or are these different on purpose?